card/mod/core/spec/set/all/assign_attributes_spec.rb

```diff
@@ -1,314 +1,37 @@
 # -*- encoding : utf-8 -*-
 
-module RenameMethods
-  def name_invariant_attributes card
-    {
-      content:     card.content,
-      # updater_id:  card.updater_id,
-      # revisions:   card.actions.count,
-      referers: card.referers.map(&:name).sort,
-      referees:    card.referees.map(&:name).sort,
-      descendants:  card.descendants.map(&:id).sort
-    }
-  end
-
-  def assert_rename card, new_name
-    attrs_before = name_invariant_attributes(card)
-    actions_count_before = card.actions.count
-    card.name = new_name
-    card.update_referers = true
-    card.save!
-    expect(card.actions.count).to eq(actions_count_before + 1)
-    assert_equal attrs_before, name_invariant_attributes(card)
-    assert_equal new_name, card.name
-    assert Card[new_name]
-  end
-
-  def card name
-    Card[name].refresh || raise("Couldn't find card named #{name}")
-  end
-end
-
 RSpec.describe Card::Set::All::AssignAttributes do
```

Comment:
All the tests below have nothing to do with `Card::Set::All::AssignAttributes`.
Some of them test the same behaviour that is already tested in `fetch_spec.rb`. 
I removed them and refactored the other and moved them to the right place.

Reply:
thank you.